code/ui/.storybook/preview.tsx

```diff
@@ -157,16 +157,6 @@ export const decorators = [
     ) : (
       <Story />
     ),
-  /**
-   * This decorator adds Symbols that the sidebar icons references.
-   * Any sidebar story that uses the icons must set the parameter withSymbols: true .
-   */
-  (Story, { parameters: { withSymbols } }) => (
-    <>
-      {withSymbols && <Symbols icons={['folder', 'component', 'document', 'bookmarkhollow']} />}
-      <Story />
-    </>
-  ),
```

Comment:
It looks like you've replaced this with adding `<IconSymbols />` manually to the stories that need it. I don't think that's better, this solution was more elegant given the amount of stories that needed this, so I suggest you bring this back.

Reply:
Since we are only using symbols on the sidebar, it make sense to keep it local for now. It was very confusing to me at first to see that this parameter was global. As discussed, we would keep it local for now.